core/src/execution/datafusion/expressions/cast.rs

```diff
@@ -174,6 +199,7 @@ impl PhysicalExpr for Cast {
         Ok(Arc::new(Cast::new(
```

Comment:
Is it safe to take the first element of the array without checking the size? I have seen in other part of the codebases safe-checks on size of arrays before taking only one element and ignoring the others

```suggestion
        match children.len() {
            1 => Ok(Arc::new(Cast::new(
                children[0].clone(),
                self.data_type.clone(),
                self.timezone.clone(),
            ))),
            _ => internal_err!("Cast should have exactly one child"),
        }
```

Reply:
Thanks. Yes, it would be good to check that cast has only one child. I will review this soon.